src/hearings/containers/request-hearing/hearing-edit-summary/hearing-edit-summary.component.ts

```diff
@@ -221,12 +221,12 @@ export class HearingEditSummaryComponent extends RequestHearingPageFlow implemen
 
     if (!!this.hearingRequestMainModel?.partyDetails) {
       partyDetailsModels = [...this.hearingRequestMainModel.partyDetails];
-      partyDetailsModels.sort(this.compareParties);
+      partyDetailsModels.sort((a, b) => this.defaultStringSort(a.partyID, b.partyID));
     }
 
     if (!!this.hearingRequestToCompareMainModel?.partyDetails) {
       partyDetailsCompareModels = [...this.hearingRequestToCompareMainModel.partyDetails];
-      partyDetailsCompareModels.sort(this.compareParties);
+      partyDetailsCompareModels.sort((a, b) => this.defaultStringSort(a.partyID, b.partyID));
```

Comment:
Well I meant more like partySort(partyDetails(Compare)Models) but this is good too